_posts/2022-01-20-saying-no.md

```diff
@@ -0,0 +1,43 @@
+I've developed a bit of a reputation inside and outside work as being pretty good with "saying no" and setting boundaries. In this post, I will help you do so, too.
+
+### Why say "no"
+
+I've been consuming a lot of Brené Brown's [podcasts](https://brenebrown.com/podcasts/) and read [one of her books](https://brenebrown.com/book/atlas-of-the-heart/) recently, and they've got me thinking about what's at the root of being able to say "no" regularly and well.
+
+[![BRAVING Trust]({{ '/images/a/braving.jpg' | absolute_url }})](https://www.linkedin.com/pulse/braving-trust-brené-brown/)
+
+Brown uses the [BRAVING acronym](https://www.linkedin.com/pulse/braving-trust-brené-brown/) to refer to the elements of Trust. Out of these, the ones that jump out to me for saying no are boundaries, reliability and integrity. To meaningfully trust others on these axes, you cannot be someone who says yes to every request. In doing so, you maintain no boundaries, professionally or personally, and will not be a reliable coworker.
+
+To the software engineers reading this, I'm sure we've all worked with some APIs that have been great and some that have been terrible. An API with inconsistent and unpredictable behaviour with the same inputs is terrible. A poorly documented API is terrible. An API that attempts to accommodate your request and consistently times out rather than failing immediately is terrible.
+
+Humans are like APIs in this regard. A bad API for your coworkers is to be someone who gives radically different estimates for the same work based on whether you want to impress the person or not. A bad API for your coworkers is when you state you're on holiday or sick but still appear in conversations and meetings anyway. A bad API for your coworkers is when you tell them a 19:00 meeting is fine, but you're hangry and irritable because you've missed dinner and your children's bedtime.
```

Comment:
Just thinking of rewording this since I read it a few times. Also, this follows the pattern of the two other examples:
```suggestion
Humans are like APIs in this regard. A bad API for your coworkers is when you're someone who gives radically different estimates for the same work based on whether you want to impress the person or not. A bad API for your coworkers is when you state you're on holiday or sick but still appear in conversations and meetings anyway. A bad API for your coworkers is when you tell them a 19:00 meeting is fine, but you're hangry and irritable because you've missed dinner and your children's bedtime.
```